utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -48,17 +49,19 @@ pub struct ThreadStats {
     classes: HashSet<&'static str>,
     in_progress_since: Option<Instant>,
     last_check: Instant,
+    c_mem: ByteSize,
 }
 
 impl ThreadStats {
     fn new() -> Self {
         Self {
-            stat: HashMap::new(),
-            cnt: 0,
+            stat: Default::default(),
+            cnt: Default::default(),
             time: Duration::default(),
-            classes: HashSet::new(),
-            in_progress_since: None,
+            classes: Default::default(),
+            in_progress_since: Default::default(),
             last_check: Instant::now(),
```

Comment:
Lol, I *hate* when you could've derived default if not for a singly pesky field )